live/app/router.js

```diff
@@ -69,6 +68,9 @@ Router.map(function() {
     this.route('create-assessment', { path: '/codecampagnepix' });
     this.route('skill-review', { path: '/codecampagnepix/resultats/:assessment_id' });
   });
+
+  // this route must always be the last one
```

Comment:
Cette route gère toutes les erreurs ? ou seulement les 404 ?
Anyway, ça mérite un commentaire un peu plus de détails je pense.

Genre:
`// XXX: this route is used for any request that did not match any of the previous routes. SHOULD ALWAYS BE THE LAST ONE`